README.md

```diff
@@ -1,85 +1,117 @@
+                        the broadcast transaction(s). Use TRACE, DEBUG, or
+                        INFO4 to see connection failures. (default: INFO)
+  --proxy PROXY, -p PROXY
+                        SOCKS5 proxy to connect through. Set to `None` to not
+                        use a proxy. (default: 127.0.0.1:9050)
+  --proxyrandomize, --no-proxyrandomize
+                        If SOCKS5 proxy is defined, assume it is TOR and use
+                        stream isolation. (default: True)
+  --timeout TIMEOUT, -t TIMEOUT
+                        Number of seconds to wait before disconnecting from
+                        nodes (default: 30)
+  --network NETWORK     Network to connect to (mainnet, regtest, testnet).
+                        This also determines the default port (default:
+                        mainnet)
+  --nodes NODES         List of nodes to connect to, denoted either host or
+                        host:port, separated by commas. If None and `nodes-
+                        file` is also None, DNS seeds will be used to populate
+                        the node list. (default: None)
+  --nodes-file NODES_FILE, -n NODES_FILE
+                        Read list of nodes from file (either host or
+                        host:port, separated one per line) (default: None)
+  --max-nodes MAX_NODES
+                        Max number of nodes to use in the node list. Set to 0
+                        to select all nodes. (default: 35)
+  --tx-file TX_FILE, -r TX_FILE
+                        Read list of transactions from file (encoded as hex,
+                        separated one per line) (default: None)
+  --QR-scan             Scan a QR code of a signed transaction from Sparrow
+                        Wallet. Note: This option is not compatible with
+                        passing transactions on the command line or when using
+                        a `tx-file`. (default: False)
+  --camera-id CAMERA_ID
+                        Camera id of the camera that you would like to use to
+                        scan a QR code with. (default: 0)
+  --camera-resolution CAMERA_RESOLUTION
+                        Camera resolution that you would like to use to scan a
+                        QR code with. (default: 1280x720)
+  --camera-mirrored     Mirror the camera when scanning QR codes. (default:
+                        False)
+  --yes, -y             Do not prompt to review and confirm transactions
+                        before submitting. (default: False)
+```
+
+The tool will connect to nodes and announce the transactions. If the
 nodes subsequently request them within the timeout, they are sent.
 
 The return status of the program will be 0 if at least one node requested the transaction, and 1
 otherwise.
 
-Example w/ Bitcoin Core
--------------------------
-
-- Send the transaction as normal, either e.g. through RPC `sendtoaddress` or the GUI
```

Comment:
Any reason to remove this example usage? Using it with bitcoin core's wallet is probably still the main application.

Reply:
We could update that to the new CLI argument format and put that back in. There was also some mix of mainnet and testnet used in that example and I didn't want to do a mainnet example, which many people would be expecting. I was just lazy I guess trying to figure out the best way to re-do it.